Formula/kamel.rb

```diff
@@ -0,0 +1,40 @@
+class Kamel < Formula
+  desc "Apache Camel K CLI"
+  homepage "https://camel.apache.org/"
+
+  url "https://github.com/apache/camel-k.git",
+    :tag      => "0.3.4",
+    :revision => "c47fb2c85e89852f0fd111d1662f57917030ced5"
+  head "https://github.com/apache/camel-k.git"
+
+  depends_on "go" => :build
+  depends_on "openjdk"
+
+  def install
+    # Build from source & Install
+    system "make"
+    bin.install "kamel"
+
+    # All that README/LICENSE/NOTES/CHANGELOG stuff
+    prefix.install_metafiles
+
+    # Install bash completion
+    output = Utils.popen_read("#{bin}/kamel completion bash")
+    (bash_completion/"kamel").write output
+
+    # Install zsh completion
+    output = Utils.popen_read("#{bin}/kamel completion zsh")
+    (zsh_completion/"_kamel").write output
+  end
+
+  test do
+    run_output = shell_output("#{bin}/kamel 2>&1")
+    assert_match "Apache Camel K is a lightweight", run_output
+
+    bash_completion_output = shell_output("#{bin}/kamel completion bash 2>&1")
+    assert_match "# bash completion for kamel", bash_completion_output
+
+    zsh_completion_output = shell_output("#{bin}/kamel completion zsh 2>&1")
+    assert_match "#compdef kamel", zsh_completion_output 
```

Comment:
These tests aren't really useful since they're already part of the install block as well

Reply:
completion tests have been removed